src/react/src/components/ClaimedFacilitiesList.jsx

```diff
@@ -26,50 +28,71 @@ function ClaimedFacilitiesList({
     error,
     getClaimed,
     clearClaimed,
+    userHasSignedIn,
 }) {
+    const TITLE = 'My Claimed Facilities';
     useEffect(() => {
         getClaimed();
 
         return clearClaimed;
```

Comment:
The cleanup function should be a function, not just a reference:
`return () => clearClaimed();`

Reply:
But clearClaimed is already function? Why to wrap it into closure one more time?

Reply:
`clearClaimed` is just a function reference but not an actual cleanup function like `clearClaimed()` (i.e., it isn't invoking).